farms/static/css/snek.css

```diff
@@ -0,0 +1,70 @@
+.snek {
+  max-width: 100%;
+  display: flex;
+  transition: 0.5s all ease-in-out;
+}
+
+.snek--small {
+  width: 10em;
+}
+.snek--medium {
+  width: 20em;
+}
+.snek--big {
+  width: 30em;
+}
+
+.snektion {
+  width: 5%;
+  display: inline-block;
+  margin-left: -2.25%;
+  transition: 0.5s all ease-in-out;
+}
+
+.snektion:first-of-type {
+  margin-left: 0;
+}
+
+.snektion--start, .snektion--end {
+  width: 8%;
+  display: inline-block;
+}
+
+.snektion--u {
+  align-self: flex-end;
+}
+
+.snektion--n {
+  align-self: flex-start;
+}
+
+.snektion .snek-body {
+  animation: one-animated 5s step-end infinite;
+}
+.snektion:nth-of-type(5n+1) .snek-body {
+  animation-delay: -4s;
+}
+.snektion:nth-of-type(5n+2) .snek-body {
+  animation-delay: -3s;
+}
+.snektion:nth-of-type(5n+3) .snek-body {
+  animation-delay: -2s;
+}
+.snektion:nth-of-type(5n+4) .snek-body {
+  animation-delay: -1s;
+}
+:root {
+  --purple: #5b57a6;
+  --green: #00b25a;
+  --orange: #f3703a;
+  --yellow: #ffc519;
+  --red: #e11d43;
+}
```

Comment:
Probably best practice to place variables at the start of the file? doesn't really need to be changed though as it works and is great. In future i would like to move this to a separate variables.css file, imported before any other CSS files so i can access the same var's in the future.